modules/traefik/deployment.tf

```diff
@@ -9,7 +9,10 @@ resource "kubernetes_deployment" "traefik" {
   spec {
     replicas = var.instances
     strategy {
-      type = "Recreate"
+      rolling_update {
+        max_unavailable = var.instances
```

Comment:
I think this only works when `instances == 1`

Reply:
Changed to 1 per our discussion via Google. Each pod will have its own PV or we'll be using non-PV storage in the future.